README.md

```diff
@@ -138,6 +138,23 @@ ex:
       --no-proc-info        Display workers process information in header.
       --refresh REFRESH     Refresh rate. Values: 0.5, 1, 2, 3, 4, 5 (default: 2).
 
+## Configuration
+
+`pg_activity` may be configured through configuration file, in [INI format][],
+read from `${XDG_CONFIG_HOME:~/.config}/pg_activity.conf` or
+`/etc/pg_activity.conf` in that order. Command-line options may override
```

Comment:
Should we specify that it's exclusive ?

Reply:
It's not "exclusive"; rather command-line options take precedence.

Reply:
I mean it's `/etc/pg_activity.conf` OR `~user/.config/pg_activity.conf`.

Reply:
That's how it's written already: "read from `${XDG_CONFIG_HOME:~/.config}/pg_activity.conf` or `/etc/pg_activity.conf`". Seems explicit enough to me.

Reply:
I first understood it as either. But it's probably my English acting up.